### PR TITLE
Add schedule operator to azimuth-ops

### PR DESCRIPTION
**Pull request by assumptionsandg**

### CODE REVIEW

File: roles/azimuth_schedule_operator/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+# The chart to use
+azimuth_schedule_operator_chart_repo: https://stackhpc.github.io/azimuth-schedule-operator
+azimuth_schedule_operator_chart_name: azimuth-schedule-operator
+azimuth_schedule_operator_chart_version:
```
Comment (JohnGarbutt) [inline roles/azimuth_schedule_operator/defaults/main.yaml:5]: Try: 0.1.0-dev.0.add-schedule-delete.18

File: roles/azimuth_schedule_operator/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    chart_version: "{{ azimuth_schedule_operator_chart_version }}"
+    release_namespace: "{{ azimuth_schedule_operator_release_namespace }}"
+    release_name: "{{ azimuth_schedule_operator_release_name }}"
+    release_values: "{{ azimuth_schedule_operator_release_values }}"
+    atomic: true
+    create_namespace: true
+    wait: true
+    wait_timeout: "{{ azimuth_schedule_operator_wait_timeout }}"
+
+- name: Install Azimuth schedule templates
```
Comment (JohnGarbutt) [inline roles/azimuth_schedule_operator/tasks/main.yaml:16]: We don't have any templates for the schedule one, we can drop this but I think

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]
Looks like a good start

File: playbooks/deploy.yml [hunk 3]
```diff
@@ -26,6 +26,8 @@
       when: azimuth_apps_enabled or azimuth_clusters_enabled
     - role: stackhpc.azimuth_ops.azimuth_caas_operator
       when: azimuth_clusters_enabled
+    - role: stackhpc.azimuth_ops.azimuth_schedule_operator
+      when: azimuth_clusters_enabled
```
Comment (JohnGarbutt) [inline playbooks/deploy.yml:30]: azimuth_kubernetes_enabled or azimuth_clusters_enabled

File: roles/azimuth_schedule_operator/defaults/main.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+# The chart to use
+azimuth_schedule_operator_chart_repo: https://stackhpc.github.io/azimuth-schedule-operator
+azimuth_schedule_operator_chart_name: azimuth-schedule-operator
+azimuth_schedule_operator_chart_version: 0.1.0-dev.0.add-schedule-delete.26
```
Comment (JohnGarbutt) [inline roles/azimuth_schedule_operator/defaults/main.yaml:5]: lets make a real release of this first.

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]

**Comment by mkjpryor** [comment]
Add to dependency automation here: https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/main/.github/workflows/update-dependencies.yml

File: roles/azimuth_schedule_operator/defaults/main.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+azimuth_schedule_operator_release_overrides: {}
+azimuth_schedule_operator_release_values: >-
+  {{-
+    azimuth_schedule_operator_release_defaults |
+      combine(azimuth_schedule_operator_release_overrides, recursive = True)
+  }}
+
+# The timeout to wait for schedule operator to become ready
+azimuth_schedule_operator_wait_timeout:
```
Comment (mkjpryor) [inline roles/azimuth_schedule_operator/defaults/main.yaml:26]: Do we not want to set this to something? Everything else uses `10m`.

**Review by mkjpryor — CHANGES_REQUESTED** [review]
Just one small thing. Otherwise looks good.

File: roles/azimuth_schedule_operator/defaults/main.yaml (hunk 5 above)
Comment (assumptionsandg) [reply roles/azimuth_schedule_operator/defaults/main.yaml]: Done

**Review by mkjpryor — APPROVED** [review]
LGTM. Test failures are not related so merging anyway.

**Pull request merged**